src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceDefault.java

```diff
@@ -63,10 +63,10 @@ public int getConstructor(Class<?>... parameterTypes)
     public static class IntEnumValueType<E extends Enum<E> & IntEnum> {
     }
 
-    public static class IntEnumValueType2<E extends Enum<E>& IntEnum> { // violation
+    public static class IntEnumValueType2<E extends Enum<E>& IntEnum> { // violation ''&' .*'
```

Comment:
message is too cryptic.
Lets think on line wrapping and chaging names in this line to let message be readable.

same for other lines.

Reply:
Can i change them to

```java
public static class EnumVal<E extends Enum<E>& IntEnum> { // violation ''&' is not preceded .*'

public static class EnumVal2<E extends Enum<E>  & IntEnum> { // violation ''>' is followed .*'
 ```
 
 The line length will be 100 and 99

Reply:
In 

```
final Supplier<Optional<Integer>> function2 = Optional::<Integer> empty; // violation ''>' .*'
```

Can be 

```
final Supplier<Optional<Integer>> f2 = Optional::<Integer>  // violation ''>' is followed .*'
empty;
```

And len will be 98

Reply:
try to remove `public static ` this Check does not care about them.

Reply:
> try to remove `public static ` this Check does not care about them. We need to remove or shorten all that does not matter for current Check/validation.

Great and the other we can shorten the name and remove final

```java
Supplier<Optional<Integer>> f2 = Optional::<Integer> empty; // violation ''>' is followed .*'
```